AIM/AIM/Data/Gateways/XMLOnAirInfoGateway.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  XMLOnAirInfoGateway.swift
+//  AIM
+//
+//  Created by Oskar Szydlowski on 01.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+import Foundation
+
+typealias OnAirInfoCompletion = (OnAirInfo) -> ()
+
+class XMLOnAirInfoGateway: NSObject, XMLParserDelegate {
```

Comment:
This should be hidden behind protocol. Also outside world shouldn't know about any parsing, i.e `parse(completion)` should be renamed